src/gui/mzroll/superSlider.h

```diff
@@ -0,0 +1,86 @@
+#ifndef SUPERSLIDER_H
+#define SUPERSLIDER_H
+#pragma once
+
+#include <QWidget>
+#include <QPainter>
+#include <QMouseEvent>
+
+class RangeSlider : public QWidget
```

Comment:
SuperSlider is a nice name. But then if the name of the class within this file is `RangeSlider`, then the filename and include guards should have the same names. Please chose one or the other. I like `RangeSlider` better - conveys exactly what it is.